specification/paths/CollectionTimeSlots.json

```diff
@@ -0,0 +1,101 @@
+        "description": "Country code in ISO 3166-1 alpha-2 format.",
+        "required": true,
+        "type": "string"
+      },
+      {
+        "name": "postal_code",
+        "in": "query",
+        "description": "Postal code of the address to search nearby.",
+        "required": true,
+        "type": "string"
+      },
+      {
+        "name": "date_from",
+        "in": "query",
+        "description": "Date string in ISO 8601 format or unix timestamp.",
+        "required": true,
+        "type": [
+          "string",
+          "number"
+        ]
+      },
+      {
+        "name": "date_to",
+        "in": "query",
+        "description": "Date string in ISO 8601 format or unix timestamp.",
+        "required": true,
+        "type": [
+          "string",
+          "number"
+        ]
+      }
+    ],
+    "responses": {
+      "200": {
+        "description": "Retrieved the locations.",
+        "schema": {
+          "type": "object",
+          "required": [
+            "data",
+            "meta",
+            "links"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "data": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/CollectionTimeSlot"
+              }
+            },
+            "meta": {
+              "$ref": "../includes/pagination-meta.json"
+            },
+            "links": {
+              "type": "object",
+              "required": [
+                "self",
+                "first",
+                "last"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "self": {
+                  "type": "string",
+                  "example": "https://api.carrier.com/collection-time-slots?country_code=NL&postal_code=1234AB&date_from=1648110600&date_to=1648114200&page[number]=3&page[size]=30"
+                },
+                "first": {
+                  "type": "string",
+                  "example": "https://api.carrier.com/collection-time-slots?country_code=NL&postal_code=1234AB&date_from=1648110600&date_to=1648114200&page[number]=1&page[size]=30"
+                },
+                "prev": {
+                  "type": "string",
+                  "example": "https://api.carrier.com/collection-time-slots?country_code=NL&postal_code=1234AB&date_from=1648110600&date_to=1648114200&page[number]=2&page[size]=30"
+                },
+                "next": {
+                  "type": "string",
+                  "example": "https://api.carrier.com/collection-time-slots?country_code=NL&postal_code=1234AB&date_from=1648110600&date_to=1648114200&page[number]=4&page[size]=30"
+                },
+                "last": {
+                  "type": "string",
+                  "example": "https://api.carrier.com/collection-time-slots?country_code=NL&postal_code=1234AB&date_from=1648110600&date_to=1648114200&page[number]=13&page[size]=30"
```

Comment:
Have these links always said https://api.carrier.com?

Reply:
Yes, for every endpoint that returns paginated stuff (statuses, pudo and now this one). We could change them all to `https://carrier.myparcel.com` or `https://microservice` or omit the domain and start with the `/` (absolute URL)?

Reply:
Eh, let's leave it like this then. It's just the microservice builders that have to do with it anyways.